divHretention/plotting/plot_along_divertor.py

```diff
@@ -250,3 +250,21 @@ def __init__(self, **kwargs):
         super().__init__(
             quantities=quantities,
             **kwargs)
+
+
+def plot_inv_with_uncertainty(x, y, sigma, alpha_fill=0.3, **kwargs):
+    """Plots the inventory along the divertor with the associated uncertainity
+    filled
+
+    Args:
+        x ([type]): Arc length (m) along the divertor
+        y ([type]): Inventory per unit thickness (H/m) 
+        sigma ([type]): Relative error
+        alpha_fill (float, optional): Shows the boundaries of uncertainty. Defaults to 0.3.
+    """
+    line, = plt.plot(x, y, **kwargs)
+    plt.fill_between(
+        x,
+        10**(2*sigma + np.log10(y)),
+        10**(-2*sigma + np.log10(y)),
```

Comment:
It would be nice if the 2 here could be a user parameter

Reply:
![image](https://user-images.githubusercontent.com/40028739/123059911-88834d80-d40a-11eb-84e7-653ea9a449d4.png)
Like 2*sigma corresponds to a 95% confidence interval.
Maybe users will want to plot the 60% confidence interval instead.